crates/biome_js_analyze/src/aria_analyzers/a11y/no_aria_unsupported_elements.rs

```diff
@@ -120,4 +122,35 @@ impl Rule for NoAriaUnsupportedElements {
             }),
         )
     }
+
+    fn action(_ctx: &RuleContext<Self>, _state: &Self::State) -> Option<JsRuleAction> {
+        let element = _ctx.query();
+        let mut mutation = _ctx.root().begin();
+        let mut removed_attribute = "role".to_string();
+
+        element.attributes().iter().for_each(|attribute| {
+            let attribute = attribute.as_jsx_attribute().unwrap();
+            let attribute_name = attribute
+                .name()
+                .ok()
+                .unwrap()
+                .as_jsx_name()
+                .unwrap()
+                .value_token()
+                .ok()
+                .unwrap();
+            let attribute_name = attribute_name.to_string();
+            removed_attribute = attribute_name;
```

Comment:
converted `remove_attribute` to `String` due to the assigning.

Reply:
Just to be sure: If you remove all `to_string()` calls (line 129 and line 134), do the compiler complain?

Reply:
Yeah it complains, there is a type difference between them.